plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployCommandHandler.java

```diff
@@ -37,15 +38,23 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
     try {
       IProject project = helper.getProject(event);
       if (project != null) {
-        final IProject project1 = project;
-        StandardDeployJob deploy = new StandardDeployJob(new ProjectToStagingExporter(), defaultStagingDir, project1);
+        String now = Long.toString(System.currentTimeMillis());
+        StandardDeployJob deploy =
+            new StandardDeployJob(new ExplodedWarPublisher(),
+                                  new StandardProjectStaging(),
+                                  new AppEngineProjectDeployer(),
+                                  getTempDir().append(now),
+                                  project);
         deploy.schedule();
```

Comment:
I defer that change for a future PR: https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/issues/384